Makefile

```diff
@@ -1,12 +1,16 @@
-.PHONY: unit e2e-test
+.PHONY: test
```

Comment:
not sure what you were trying to do here, but i don't think this is right:
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

if you want to add a new test target that invokes other targets, it would look like:

```
test:  test-unit test-e2e
```

Reply:
i did a quick read on `.PHONY` and forgot to change this back, but I think it was wrong to begin with anyway.

tl;dr
`.PHONY` needs to include any targets that might also be files. I don't think we would have a file called `e2e-test` or `test-unit`, so this `.PHONY` is probably irrelevant.